pkg/scheduler/metrics/metrics.go

```diff
@@ -110,8 +110,11 @@ var (
 	FrameworkExtensionPointDuration *metrics.HistogramVec
 	PluginExecutionDuration         *metrics.HistogramVec
 
-	PermitWaitDuration    *metrics.HistogramVec
+	PermitWaitDuration *metrics.HistogramVec
+	// Deprecated: Please use SchedulerCacheSize instead.
+	// TODO(googs1025): CacheSize will be deprecated in the future release cycle.
```

Comment:
```suggestion
	// Deprecated: SchedulerCacheSize is deprecated at v1.33,
	// and will be removed at v1.34. Please use SchedulerCacheSize instead.
```

Reply:
Please create an GitHub issue instead of TODO comment here.

Reply:
got this. i will create an issue to track this.